src/main/java/org/myrobotlab/service/InMoov2.java

```diff
@@ -125,35 +66,35 @@ public void execScript(String someScriptName) {
     }
   }
 
-  /**
-   * This method will load a python file into the python interpreter.
-   */
+	/**

```

Comment:
@Grog check your auto-format settings.. i think they've switched to tabs not spaces .. at least for this file.